app/views/pages/homepage.html.erb

```diff
@@ -1,84 +1,124 @@
-<div class="usa-grid homepage-hero">
-  <h2>While moving is never easy, <b>move.mil</b> makes your experience just a little bit better.</h2>
-</div>
-
-<div class="usa-grid">
-  <aside class="usa-width-one-third">
-    <div class="dps-login-section">
-      <p><b>Defense Personal Property System (DPS)</b> is your portal to move scheduling and the claims process.</p>
-      <p><%= link_to raw("Log In to #{abbr_tag('dps')}"), 'https://eta.sddc.army.mil/ETASSOPortal/SSO/AppLogin.aspx?LoginType=portal&amp;SSOAppID=DPS', class: 'usa-button' %></p>
-      <p class="usa-text-small">New to <%= abbr_tag('dps') %>? <%= link_to 'Create your account', 'https://www.move.mil/dod/first_time_users/browser_compatibility.cfm' %></p>
-    </div>
-  </aside>
-
-  <div class="usa-width-two-thirds">
-    <h2 class="homepage-content-title">Your Moving Guide</h2>
+<section class="usa-hero"></section>
```

Comment:
What's the purpose of this empty `<section>`?

Reply:
This is for the banner image. Not sure if there's a better way to do this. Although I'm going to change this now to include the nav to do the transparent nav bar thing in @lfantano's design.